internal/cmd/schema-registry/command_config_describe_onprem.go

```diff
@@ -10,12 +10,12 @@ import (
 	pversion "github.com/confluentinc/cli/internal/pkg/version"
 )
 
-func (c *configCommand) newDescribeCommandOnPrem() *cobra.Command {
+func (c *command) newConfigDescribeCommandOnPrem() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:         "describe",
 		Short:       "Describe top-level or subject-level schema compatibility.",
-		Args:        cobra.MaximumNArgs(0),
-		RunE:        c.onPremDescribe,
+		Args:        cobra.NoArgs,
+		RunE:        c.configDescribeOnPrem,
```

Comment:
other commands like kafka are called `onpremOperation` as well. i think it's easier to read with `onprem` prefix.

Reply:
Can you give me an example? I think all of the function names should be suffixed with "OnPrem".

Reply:
https://github.com/confluentinc/cli/blob/ac394e7f40f93c2ee424d7e8acf26269d5e12d4b/internal/cmd/kafka/command_cluster_list_onprem.go#L31

Reply:
Thanks, I'll standardize these soon!

Reply:
(in a different PR)